doc/changelog_user_guide.md

```diff
@@ -0,0 +1,396 @@
+| `include.contributors`                | `string[]` | The list of contributors that are _required_ for the author of a pull-request to be listed in the changelog. By default this list is empty.
+| `exclude.labels`                      | `string[]` | The list of labels that if present on a pull-request will exclude the pull-request from being listed in the changelog. By default and if `defaults = true`, this list contains `skip-release-notes`.
+| `exclude.contributors`                | `string[]` | The list of contributors that if present on a pull-request will exclude the pull-request from being listed in the changelog.
+| `body_template`                       | `string`   | A template of the body of the release notes. The default is defined below.
+| `template_properties`                 | `map<string, object>`  | Allow to define user properties that are made accessible in all templates through the `properties` variable. See usage in following sections.
+| `[[changelog.autolabeler]]`           | `autolabeler` | The list of autolabelers configured. This is described in more details in the following sections.
+| `[[changelog.category]]`              | `category`    | The list of categories configured. This is described in more details in the following sections.
+
+The default of the `body_template` is the following [Scriban](https://github.com/scriban/scriban) template with `markdown`:
+
+```md
+# Changes
+
+{{ changes }}
+
+**Full Changelog**: {{ url_full_changelog_compare_changes }}
+```
+### 1.3 Categories
+
+Categories allow to define how the changes are organized within your release notes. By default if `defaults = true` is kept `true`, the following categories will be define.
+
+| `[[changelog.category]]`              | Type       | Description                |
+|---------------------------------------|------------|----------------------------|
+| `title`                               | `string`   | Defines the markdown title that will be used for this category in the release notes.
+| `labels`                              | `string[]` | A list of pull-request labels that are associated with this category. Each pull-request that will have this label defined in this list will belong to this category.
+| `exclude.labels`                      | `string[]` | A list of pull-request labels that, if defined on a pull-request will exclude the pull-request to be part of this category.
+| `exclude.contributors`                | `string[]` | A list of pull-request contributors that, if are the authors in the pull-request will exclude the PR to be part of this category.
+| `order`                               | `int`      | An integer that defines the order of the category when it will be displayed in the release notes. A lower the value is, the higher in the list in the release notes the category will be displayed. If this value is not defined, it is assumed to be 0.
```

Comment:
**The** lower the value is, the higher in the list in the release notes the category will be displayed